config/jobs/kubernetes/sig-release/conformance-gate.yaml

```diff
@@ -0,0 +1,14 @@
+periodics:
+- interval: 4h
+  name: conformance-gate
+  decorate: true
+  spec:
+    containers:
+    - name: apisnoop-gate
+      image: zachboyofdestiny/snoopdb:gate
```

Comment:
could we avoid using a personal image here?

Reply:
@feiskyer , yep!  We just put in a change to APISnoop so that the new database it uses can be a part of the existing test-infra image builds for APISnoop.  I opened a PR for this change here: https://github.com/kubernetes/test-infra/pull/19234

When merged, I can update the container here to use this image.

Reply:
@feiskyer , This job now uses an image from `gcr.io/k8s-staging-apisnoop`.